src/database/DataTypes/Sensor.js

```diff
@@ -92,6 +92,22 @@ export class Sensor extends Realm.Object {
   get currentTemperature() {
     return this.mostRecentLog?.temperature;
   }
+
+  get isInHotBreach() {
+    const breach = this.mostRecentBreach;
+    const isOngoing = !!breach?.endTimestamp;
```

Comment:
is it ongoing if there is an end? I thought that it would be the other way round

Reply:
ah doh, nice! thanks 

0882239